test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -1204,7 +1225,7 @@ describe('emitConversationEvent()', () => {
 
     handler.conversations[conversationState.conversationId] = conversationState;
     expectedEvent.current.push(conversationState);
-    handler.emitConversationEvent('updated', conversationState, undefined);
+    handler.emitConversationEvent('updated', conversationState, undefined as any);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    handler.emitConversationEvent('updated', conversationState, undefined as unknown);
```